instructure/instructure-ui
# fix(many): fix regression where form elements without label got misaligned

- this PR fixes a regression introduced here: https://github.com/instructure/instructure-ui/pull/1729 where form elements without label got an unnecessary top margin which made them misaligned
- also updates the prop description for `messages` where needed

test plan:

- check out pagination example with number input: should not be misaligned
- check other form elements without label: should not be misaligned
